dockerfile_parse/parser.py

```diff
@@ -100,6 +107,15 @@ def __init__(self, path=None, cache_content=False, env_replace=True, parent_env=
         else:
             self.parent_env = {}
 
+    @contextmanager
+    def _open_dockerfile(self, mode):
+        if self.fileobj is not None:
+            self.fileobj.seek(0)
```

Comment:
Thanks for mentioning it. Is there any easier way instead of using this?

```python
try:
    self.fileobj.seek(0)
except IOError:
    # not seekable fileobj
    pass
```